src/tools.ts

```diff
@@ -201,6 +141,7 @@ function lockCodeUsingDecoration (
         return editorOperations
       }
 
+      editorOperations = computeNewOperationsForLockedCode(model, decorationFilter, editorOperations, withDecoration)
```

Comment:
maybe we can find a more meaningful name :thinking: like `tryIgnoreLockedCode` or something?

Reply:
[refactor: rename computeNewOperationsForLockedCode to tryIgnoreLockedCode](https://github.com/CodinGame/monaco-editor-wrapper/pull/125/commits/0759eb0e9e1235de24c289da9b02cc7e9a14580d)